pandas/core/arrays/base.py

```diff
@@ -216,6 +218,88 @@ def isna(self):
         """
         raise AbstractMethodError(self)
 
+    def tolist(self):
+        # type: () -> list
+        """Convert the array to a list of scalars."""
+        return list(self)
```

Comment:
Is this needed for fillna?

Reply:
Indirectly, via the default `fillna`.

`Series.__iter__` calls `Series.tolist`, which calls `Series._values.tolist`. We could modify that to check the type and just call `list(self._values)` for EAs. I don't have a strong preference vs. adding a default `.tolist`.

Reply:
Or directly call `values.__iter__` ? As iterating through values to make a list and then iterate again through that list sounds like a bit of overhead.

For `.tolist()` itself, it is of course consistent with Series and numpy arrays, but personally I am not sure what its value is compared to `list(values)`. I don't think you typically can implement a more efficient conversion to lists than what `list(values)` will do by default? (i.e. iterating through the values)